system/datetime-plugin/wrapper/tests/ e2e.test.ts

```diff
@@ -0,0 +1,39 @@
+import { PolywrapClient } from "@polywrap/client-js";
+import { dateTimePlugin } from "../../plugin/build";
+import path from "path"
+
+jest.setTimeout(360000);
+
+describe("e2e", () => {
+
+  let client: PolywrapClient;
+  let uri: string;
+
+  beforeAll(async () => {
+      // Create the client w/ test env configuration + the dateTimePlugin
+      client = new PolywrapClient({
+        plugins: [{
+          uri: "ens/datetime.eth",
```

Comment:
let's keep consistency in naming URI

Reply:
fixed. I updated the version from demos, not the one in integrations, and that's why this was changed.